laboratory-servlet-api-integration/src/main/java/com/wixpress/petri/laboratory/http/LaboratoryFilter.java

```diff
@@ -121,6 +134,26 @@ public void init(FilterConfig filterConfig) throws ServletException {
         startMetricsReporterScheduler(laboratoryTopology.getReportsScheduleTimeInMillis());
 
         FilterTypeIdResolver.useDynamicFilterClassLoading();
+        cache = setupKoboshiCache(context);
+    }
+
+    private ReadOnlyTimestampedLocalCache<ConductibleExperiments> setupKoboshiCache(final ServletContext context) {
```

Comment:
My readme isn't updated yet. It will be there.
I can move the koboshi stuff but this whole filter feels a bit of a mess. I'll do as you wish.